### ISSUE TITLE
Better fieldbook format for levelling

### ISSUE THREAD

**Issue by zsiki** [zsiki/GeoEasy#55]
The columns of levelling fieldbook confuse users, these are valid for total stations.
Separate fieldbook template should be created for levelling.

**Comment by zsiki** [comment]
Beside "table" type forms there should be an other for example "simple" to handle individual observations. (no station)

**Comment by drf5n** [comment]
Does the existing fieldbook format work for levelling, even if confusing? 

If it does, maybe demonstrating / documenting how it can be used for levelling would be helpful.

**Comment by zsiki** [comment]
Yes it works. You can load DNA03 or NA3000 GSI files. You should select levelling mask for the field book. You can manually input levelling data, too.

For the documentation I need help...

**Comment by zsiki** [comment]
A short description added to process levelling network to step by step documentation

**Comment by zvezdochiot** [comment]
Good work!

**Comment by drf5n** [comment]
Cool!  Thanks.

Could you add the `levelling.dmp` sample file to https://github.com/zsiki/GeoEasy/tree/master/src/demodata ?  

I tried a cut and paste of the ASCII table from the step-by-step but didn't seem to load a levelling_coo file

**Comment by zsiki** [comment]
I've added levelling.dmp file to demodata directory. There is no levelling.coo file. As you can see from the description I set manually the elevation of a point and made a free levelling network adjustment after.
